docker/pypi/wmagent/etc/wmagent_bashrc

```diff
@@ -0,0 +1,25 @@
+### Template .bashrc file to be used for the user running WMAgent
+alias agentenv='source $WMA_ENV_FILE'
+alias manage=\$WMA_MANAGE_DIR/manage
+
+alias lll="ls -lathr"
+alias ls="ls --color=auto"
+alias ll='ls -la --color=auto'
+
+alias condorq='condor_q -format "%i." ClusterID -format "%s " ProcId -format " %i " JobStatus  -format " %d " ServerTime-EnteredCurrentStatus -format "%s" UserLog -format " %s\n" DESIRED_Sites'
+alias condorqrunning='condor_q -constraint JobStatus==2 -format "%i." ClusterID -format "%s " ProcId -format " %i " JobStatus  -format " %d " ServerTime-EnteredCurrentStatus -format "%s" UserLog -format " %s\n" DESIRED_Sites'
+
+alias runningagent="ps aux | egrep 'couch|wmcore|mysql|beam'"
+alias foldersize="du -h --max-depth=1 | sort -hr"
+
+# Better curl command
+alias scurl='curl -k --cert ${WMA_CERTS_DIR}/servicecert.pem --key ${WMA_CERTS_DIR}/servicekey.pem'
+
+# Set command prompt for the running user inside the container
+export PS1="(WMAgent-\$WMA_TAG) [\u@\h:\W]\$ "
+
```

Comment:
It is a good way to load this from a file (me likes) , but I feel a line or two are missing. I thought while we were working with Andrea on the opportunistic resources we have added the line: 
```
source $WMA_DEPLOY_DIR/bin/manage-common.sh
```
which is quite helpful, because exposes  all `_*` auxiliary functions defined there, and makes it quite easy during operations to manually call stuff like `_print_settings`, _load_wmagentsecrets` , `_renew_proxy`

Reply:
and indeed, @amaltaro, those two very important lines are missing:  https://github.com/dmwm/CMSKubernetes/pull/1462/files#diff-fcb5388aefac16c54c533239a64588ccb9bec26a5ab1d818a9d190830144a239R178-R179 

```
source $WMA_ENV_FILE
source $WMA_DEPLOY_DIR/bin/manage-common.sh
```

Could you please add them at the end of the `~/.bashrc` file.

Reply:
Fixed! Thank you for catching this, Todor.